src/Elastic.Markdown/_static/custom.css

```diff
@@ -15,6 +15,14 @@ This hides the ugly one */
   display: none !important;
 }
 
+.sd-tab-content {
+  padding-top: 20px !important;
+}
+
+body {
+  background-color: #FF0000;
```

Comment:
This is screaming at me :smile_cat:

Reply:
Wait does that work for you? I can't get the CSS to work on my end and forgot about those. I'll update.